package.json

```diff
@@ -41,8 +41,11 @@
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-flow": "^7.10.4",
     "@babel/register": "^7.10.5",
+    "@playkit-js/info": "^3.3.2",
     "@playkit-js/kaltura-player-js": "latest",
+    "@playkit-js/moderation": "latest",
```

Comment:
i know you're replacing it with canary later, but reminder -  always use explicit versions here